### PR TITLE
add vee-validate@0.9.3 w/ npm auto-update

### LINKED ISSUE #9646: [Request] Add vee-validate
**Library name:** vee-validate
**Git repository url:** https://github.com/logaretm/vee-validate
**npm package url(optional):** https://www.npmjs.com/package/vee-validate
**License(s):** MIT
**Official homepage:** http://vee-validate.logaretm.com/
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by lcd78706**
Pull request for issue: #9646 
Related issue(s): #9414 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/logaretm/vee-validate
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/vee-validate
 * GitHub / Bitbucket popularity (required): 
   - Count of stars: 375
   - Count of watchers: 19
   - Count of forks: 57
 * NPM download stats (optional):
   - Downloads in the last day: 214
   - Downloads in the last week: 1071
   - Downloads in the last month: 5128

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

cc #9414, close #9646, cc @logaretm

### CODE REVIEW

**Comment by lcd78706** [comment]
@kennynaoh 
Please review this PR. Thanks!

File: ajax/libs/vee-validate/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "vee-validate",
+  "filename": "vee-validate.min.js",
+  "version": "0.9.3",
+  "description": "Simple Vue.js input validation plugin",
+  "homepage": "https://vee-validate.logaretm.com",
```
Comment (kennynaoh) [inline ajax/libs/vee-validate/package.json]: @lcd78706 Please use `http` here.

Comment (lcd78706) [inline ajax/libs/vee-validate/package.json]: @kennynaoh 
I have fixed the problem.

**Comment by kennynaoh** [comment]
LGTM, ping @cdnjs/intern2

**Comment by lcd78706** [comment]
@pvnr0082t 
I have fixed it.

**Comment by lcd78706** [comment]
@pvnr0082t 
Sorry. I installed the wrong version.
I have reinstalled and fixed it.

**Comment by lcd78706** [comment]
@kennynaoh   OK. I have minified the file.

**Comment by kennynaoh** [comment]
@lcd78706 The minified file is broken.

**Comment by kennynaoh** [comment]
@PeterDaveHello Please help review this problem, thanks!

**Comment by PeterDaveHello** [comment]
Let's just not do the minify on locales in this case since there is no minifier support es6 yet.

**Comment by lcd78706** [comment]
@kennynaoh  Please review this PR again. Thanks!

File: ajax/libs/vee-validate/package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  "name": "vee-validate",
+  "filename": "vee-validate.min.js",
+  "version": "0.9.3",
+  "description": "Simple VueJS input validation plugin",
+  "homepage": "http://vee-validate.logaretm.com",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/logaretm/vee-validate.git"
+  },
+  "ava": {
```
Comment (kennynaoh) [inline ajax/libs/vee-validate/package.json]: @lcd78706 Please remove this field.

Comment (lcd78706) [inline ajax/libs/vee-validate/package.json]: @kennynaoh  I have removed it.

**Comment by kennynaoh** [comment]
LGTM, ping @cdnjs/intern2

**Comment by pvnr0082t** [comment]
LGTM

**Pull request merged**